src/main/java/org/springframework/data/tarantool/core/mapping/BasicTarantoolPersistentProperty.java

```diff
@@ -41,10 +40,18 @@ public BasicTarantoolPersistentProperty(Property property,
 
     @Override
     public String getFieldName() {
-        Field annotationField = AnnotatedElementUtils.findMergedAnnotation(getField(), Field.class);
-
-        if (annotationField != null && StringUtils.hasText(annotationField.value())) {
-            return annotationField.value();
+        java.lang.reflect.Field field = getField();
+        Field annotationField = field.getAnnotation(Field.class);
```

Comment:
You are changing the result here. Without `findMergingAnnotation` it will not find annotations on overriden fields, for example. Please read: https://docs.spring.io/spring-framework/docs/5.0.2.RELEASE/kdoc-api/spring-framework/org.springframework.core.annotation/-annotated-element-utils/find-merged-annotation.html